ckan/lib/helpers.py

```diff
@@ -2595,3 +2595,16 @@ def sanitize_id(id_):
     ValueError.
     '''
     return str(uuid.UUID(id_))
+
+
+@core_helper
+def public_user_details():
+    return converters.asbool(config.get('ckan.public_user_details', True))
+
+
+@core_helper
+def restrict_anon(context):
```

Comment:
this doesn't make sense as a template helper, please make this a simple function in logic/auth/get.py instead